### PR TITLE
Update Manager.php

### PR DESCRIPTION
**Pull request by aleritty**
Added capability to send caption with a photo

### CODE REVIEW

File: src/Manager.php [hunk 1]
```diff
@@ -64,11 +64,12 @@ public function postMessage($message, $attachmentPath = null)
      *
      * @var string $photoPath
      */
-    public function postPhoto($photoPath)
+    public function postPhoto($photoPath,$caption=NULL)
```
Comment (naffiq) [inline src/Manager.php]: Could you please add space after comma to maintain code style? And also transform `NULL` to lowercase.

Also, you didn't add PHPDoc for new attribute, thats gonna be `@var string|null $caption`

**Review by naffiq — COMMENTED** [review]
Thanks for your contribution, caption is really what we needed! But in order to maintain repo for coming users we have to maintain code style and PHPDoc!
I would be very thankful if you complete your PR with changes proposed

**Comment by aleritty** [comment]
Sorry for that, I just thought about a very fast modification. I'll try to stick to the styling from now on!

**Review by naffiq — APPROVED** [review]
Nice! Will release in as soon as I cover it with appropriate test case

**Comment by naffiq** [comment]
@aleritty 

Released v1.0.2 which includes your updates! Thank you for contribution!

**Pull request merged**